browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -46,13 +48,17 @@ namespace {
 constexpr SkColor kHeaderButtonColor = SkColorSetRGB(0x6B, 0x70, 0x84);
 constexpr int kHeaderInset = 4;
 
-class ToggleButton : public views::Button {
+// Inherits NewTabButton in order to synchronize ink drop effect with
+// the search button. Unfortunately, we can't inherit BraveTabSearchButton
+// as NotifyClick() is marked as 'final'.
```

Comment:
If this becomes a problem in the future because chromium changes NewTabButton and breaks ToggleButton here, maybe we can find a way, even if it's a patch, to unmark NotifyClick() as final. Fine for now though.